packages/components/src/components/form/Input/Input.tsx

```diff
@@ -131,7 +133,7 @@ const Input = ({
         <Wrapper
             onMouseEnter={() => setIsHovered(true)}
             onMouseLeave={() => setIsHovered(false)}
-            withBottomPadding={bottomText === null}
+            withBottomPadding={withBottomPadding === undefined && bottomText === null}
```

Comment:
Since it's `withBottomPadding` by default, it'd add `isWithoutBottomPadding` and change the condittion here. Also, not a fan of referring to padding in here, since it's really about the `Input` having no `BottomText` – no errors, no other messages... How about `noBottomText`?

Reply:
https://github.com/trezor/trezor-suite/pull/10050/commits/4e43da5f6ad6d8c577d90a35ae11cca64a284fe3

Reply:
I'd write it as `withBottomPadding={!isWithoutBottomPadding || bottomText === null}`

and again, not having the padding is a secodary trait of not having nor not going to have `BottomText`, that's why I was thinking about some other prop name 😅